github/tests/ReplayData/Repository.testGetContentDir.txt

```diff
@@ -2,7 +2,7 @@ https
 GET
 api.github.com
 None
-/repos/jacquev6/PyGithub/contents/
+/repos/jacquev6/PyGithub/contents//
```

Comment:
Same here, is it necessary?

Reply:
Not "necessary", just an artifact of how the issue is being addressed. Please see previous response.